<?xml version="1.0" encoding="ascii"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
          "DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>copads.copads.dose_parameters</title>
  <link rel="stylesheet" href="epydoc.css" type="text/css" />
  <script type="text/javascript" src="epydoc.js"></script>
</head>

<body bgcolor="white" text="black" link="blue" vlink="#204080"
      alink="#204080">
<!-- ==================== NAVIGATION BAR ==================== -->
<table class="navbar" border="0" width="100%" cellpadding="0"
       bgcolor="#a0c0ff" cellspacing="0">
  <tr valign="middle">
  <!-- Home link -->
      <th>&nbsp;&nbsp;&nbsp;<a
        href="copads.copads-module.html">Home</a>&nbsp;&nbsp;&nbsp;</th>

  <!-- Tree link -->
      <th>&nbsp;&nbsp;&nbsp;<a
        href="module-tree.html">Trees</a>&nbsp;&nbsp;&nbsp;</th>

  <!-- Index link -->
      <th>&nbsp;&nbsp;&nbsp;<a
        href="identifier-index.html">Indices</a>&nbsp;&nbsp;&nbsp;</th>

  <!-- Help link -->
      <th>&nbsp;&nbsp;&nbsp;<a
        href="help.html">Help</a>&nbsp;&nbsp;&nbsp;</th>

  <!-- Project homepage -->
      <th class="navbar" align="right" width="100%">
        <table border="0" cellpadding="0" cellspacing="0">
          <tr><th class="navbar" align="center"
            ><a class="navbar" target="_top" href="https://github.com/mauriceling/copads">https://github.com/mauriceling/copads</a></th>
          </tr></table></th>
  </tr>
</table>
<table width="100%" cellpadding="0" cellspacing="0">
  <tr valign="top">
    <td width="100%">
      <span class="breadcrumbs">
        Package&nbsp;copads ::
        <a href="copads.copads-module.html">Package&nbsp;copads</a> ::
        Module&nbsp;dose_parameters
      </span>
    </td>
    <td>
      <table cellpadding="0" cellspacing="0">
        <!-- hide/show private -->
        <tr><td align="right"><span class="options">[<a href="javascript:void(0);" class="privatelink"
    onclick="toggle_private();">hide&nbsp;private</a>]</span></td></tr>
        <tr><td align="right"><span class="options"
            >[<a href="frames.html" target="_top">frames</a
            >]&nbsp;|&nbsp;<a href="copads.copads.dose_parameters-module.html"
            target="_top">no&nbsp;frames</a>]</span></td></tr>
      </table>
    </td>
  </tr>
</table>
<!-- ==================== MODULE DESCRIPTION ==================== -->
<h1 class="epydoc">Module dose_parameters</h1><p class="nomargin-top"><span class="codelink"><a href="copads.copads.dose_parameters-pysrc.html">source&nbsp;code</a></span></p>
<p>Parameter file for DOSE simulation. Date created: 13th September 2012 
  Licence: Python Software Foundation License version 2</p>
  <p>This file contains the default parameters for DOSE simulation.</p>

<!-- ==================== VARIABLES ==================== -->
<a name="section-Variables"></a>
<table class="summary" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr bgcolor="#70b0f0" class="table-header">
  <td colspan="2" class="table-header">
    <table border="0" cellpadding="0" cellspacing="0" width="100%">
      <tr valign="top">
        <td align="left"><span class="table-header">Variables</span></td>
        <td align="right" valign="top"
         ><span class="options">[<a href="#section-Variables"
         class="privatelink" onclick="toggle_private();"
         >hide private</a>]</span></td>
      </tr>
    </table>
  </td>
</tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#initial_chromosome" class="summary-name">initial_chromosome</a> = <code title="['0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0',\
 '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0',\
 '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0',\
 '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0',\
 '0', '0', '0', '0']">['0', '0', '0', '0', '0', '0', '0', '0', <code class="variable-ellipsis">...</code></code><br />
      Probability of number of mutations per base.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#background_mutation_rate" class="summary-name">background_mutation_rate</a> = <code title="0.1">0.1</code><br />
      Probability of mutation per base above additional_mutation_rate.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a name="additional_mutation_rate"></a><span class="summary-name">additional_mutation_rate</span> = <code title="0">0</code><br />
      Size of cytoplasm (length of list) for the ancestor organism.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#cytoplasm_size" class="summary-name">cytoplasm_size</a> = <code title="50">50</code><br />
      Maximum size of cytoplasm (length of list) allowable.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#max_cytoplasm_size" class="summary-name">max_cytoplasm_size</a> = <code title="200">200</code><br />
      If 'True', a clean cytoplasm will be used to execute the genome.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#clean_cytoplasm" class="summary-name">clean_cytoplasm</a> = <code title="True">True</code><br />
      Maximum number of codons to evaluate.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#max_codon" class="summary-name">max_codon</a> = <code title="2000">2000</code><br />
      List containing name(s) of population.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a name="population_names"></a><span class="summary-name">population_names</span> = <code title="['pop1', 'pop2']">['pop1', 'pop2']</code><br />
      Number of organism per population.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a name="population_size"></a><span class="summary-name">population_size</span> = <code title="100">100</code><br />
      Number of ecological cells in the world in (x,y,z) coordinate as 
      (world_x, world_y, world_z).
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a name="world_x"></a><span class="summary-name">world_x</span> = <code title="5">5</code>
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a name="world_y"></a><span class="summary-name">world_y</span> = <code title="5">5</code>
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#world_z" class="summary-name">world_z</a> = <code title="5">5</code><br />
      List of tuple indicating the location on ecosystem to map the 
      population(s) and will be mapped in the same order as 
      population_names.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a name="population_locations"></a><span class="summary-name">population_locations</span> = <code title="[(0, 0, 0), (4, 4, 4)]">[(0, 0, 0), (4, 4, 4)]</code><br />
      Maximum number of generations to simulate.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#maximum_generations" class="summary-name">maximum_generations</a> = <code title="500">500</code><br />
      Number of generations between each freezing/fossilization event.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#fossilized_frequency" class="summary-name">fossilized_frequency</a> = <code title="100">100</code><br />
      Proportion of population to freeze/fossilize.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#fossilized_ratio" class="summary-name">fossilized_ratio</a> = <code title="0.01">0.01</code><br />
      Dictionary containing prefix of file names for 
      freezing/fossilization.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#fossil_files" class="summary-name">fossil_files</a> = <code title="{'pop1': 'pop1', 'pop2': 'pop2'}">{'pop1': 'pop1', 'pop2': 'pop2'}</code><br />
      Number of generations between printing of reports (based on 
      Population.report function) into files.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#print_frequency" class="summary-name">print_frequency</a> = <code title="10">10</code><br />
      Dictionary containing file names for result files.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a name="result_files"></a><span class="summary-name">result_files</span> = <code title="{'pop1': 'pop1', 'pop2': 'pop2'}">{'pop1': 'pop1', 'pop2': 'pop2'}</code><br />
      Version of Ragaraja instruction set to use.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#ragaraja_version" class="summary-name">ragaraja_version</a> = <code title="0.1">0.1</code><br />
      File containing Ragaraja instruction set to be used.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a name="user_defined_instructions"></a><span class="summary-name">user_defined_instructions</span> = <code title="'ragaraja_instructions.txt'">'ragaraja_instructions.txt'</code><br />
      Number of generations between each burial/preservation of ecosystem.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a href="copads.copads.dose_parameters-module.html#eco_buried_frequency" class="summary-name">eco_buried_frequency</a> = <code title="500">500</code><br />
      Prefix of file name for ecosystem burial/preservation.
    </td>
  </tr>
<tr>
    <td width="15%" align="right" valign="top" class="summary">
      <span class="summary-type">&nbsp;</span>
    </td><td class="summary">
        <a name="eco_burial_file"></a><span class="summary-name">eco_burial_file</span> = <code title="'eco'">'eco'</code>
    </td>
  </tr>
</table>
<!-- ==================== VARIABLES DETAILS ==================== -->
<a name="section-VariablesDetails"></a>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr bgcolor="#70b0f0" class="table-header">
  <td colspan="2" class="table-header">
    <table border="0" cellpadding="0" cellspacing="0" width="100%">
      <tr valign="top">
        <td align="left"><span class="table-header">Variables Details</span></td>
        <td align="right" valign="top"
         ><span class="options">[<a href="#section-VariablesDetails"
         class="privatelink" onclick="toggle_private();"
         >hide private</a>]</span></td>
      </tr>
    </table>
  </td>
</tr>
</table>
<a name="initial_chromosome"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">initial_chromosome</h3>
  <p>Probability of number of mutations per base. For example, 0.1 means 
  10% of the chromosome will be mutated per generation.</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
['0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0',<span class="variable-linewrap"><img src="crarr.png" alt="\" /></span>
 '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0',<span class="variable-linewrap"><img src="crarr.png" alt="\" /></span>
 '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0',<span class="variable-linewrap"><img src="crarr.png" alt="\" /></span>
 '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0',<span class="variable-linewrap"><img src="crarr.png" alt="\" /></span>
 '0', '0', '0', '0']
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="background_mutation_rate"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">background_mutation_rate</h3>
  <p>Probability of mutation per base above additional_mutation_rate. No 
  mutation event will ever happen if (additional_mutation_rate + 
  additional_mutation_rate) is less than zero.</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
0.1
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="cytoplasm_size"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">cytoplasm_size</h3>
  <p>Maximum size of cytoplasm (length of list) allowable. Some Ragaraja 
  instructions can increase cytoplasms size, equivalent to cell 
  enlargement.</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
50
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="max_cytoplasm_size"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">max_cytoplasm_size</h3>
  <p>If 'True', a clean cytoplasm will be used to execute the genome. If 
  'False', the cytoplasm will be reused in the next generation.</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
200
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="clean_cytoplasm"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">clean_cytoplasm</h3>
  <p>Maximum number of codons to evaluate. Therfore, the maximum evaluated 
  genome is 3*max_codon bases.</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
True
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="max_codon"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">max_codon</h3>
  <p>List containing name(s) of population. The number of names = the 
  number of populations to create.</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
2000
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="world_z"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">world_z</h3>
  <p>List of tuple indicating the location on ecosystem to map the 
  population(s) and will be mapped in the same order as population_names. 
  For example, if population_names = ['pop1', 'pop2'] and 
  population_locations = [(0,0,0), (4,4,4)], population 'pop1' will be in 
  (0,0,0) and population 'pop2' will be in (4,4,4).</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
5
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="maximum_generations"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">maximum_generations</h3>
  <p>Number of generations between each freezing/fossilization event. 
  Freezing/fossilization event is similar to glycerol stocking in 
  microbiology.</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
500
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="fossilized_frequency"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">fossilized_frequency</h3>
  <p>Proportion of population to freeze/fossilize. If the population size 
  or the preserved proportion is below 100, the entire population will be 
  preserved.</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
100
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="fossilized_ratio"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">fossilized_ratio</h3>
  <p>Dictionary containing prefix of file names for freezing/fossilization.
  The preserved sample will be written into a file with name in the 
  following format - &lt;prefix&gt;_&lt;generation count&gt;_&lt;sample 
  size&gt;.gap</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
0.01
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="fossil_files"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">fossil_files</h3>
  <p>Number of generations between printing of reports (based on 
  Population.report function) into files. The result file format is &lt;UTC
  date time stamp&gt; &lt;current generation count&gt; &lt;output from 
  Population.report function&gt;.</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
{'pop1': 'pop1', 'pop2': 'pop2'}
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="print_frequency"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">print_frequency</h3>
  <p>Dictionary containing file names for result files. File name will be 
  &lt;file names&gt;.result</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
10
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="ragaraja_version"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">ragaraja_version</h3>
  <p>File containing Ragaraja instruction set to be used. This option is 
  only effective when ragaraja_version = 0. Format of file is 
  &lt;instruction&gt;={Y|N} where &quot;Y&quot; = instruction to be used 
  and &quot;N&quot; = instruction not to be used.</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
0.1
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<a name="eco_buried_frequency"></a>
<div>
<table class="details" border="1" cellpadding="3"
       cellspacing="0" width="100%" bgcolor="white">
<tr><td>
  <h3 class="epydoc">eco_buried_frequency</h3>
  <p>Prefix of file name for ecosystem burial/preservation. The preserved 
  ecosystem will be written into a file with name in the following format -
  &lt;prefix&gt;_&lt;generation count&gt;.eco</p>
  <dl class="fields">
  </dl>
  <dl class="fields">
    <dt>Value:</dt>
      <dd><table><tr><td><pre class="variable">
500
</pre></td></tr></table>
</dd>
  </dl>
</td></tr></table>
</div>
<br />
<!-- ==================== NAVIGATION BAR ==================== -->
<table class="navbar" border="0" width="100%" cellpadding="0"
       bgcolor="#a0c0ff" cellspacing="0">
  <tr valign="middle">
  <!-- Home link -->
      <th>&nbsp;&nbsp;&nbsp;<a
        href="copads.copads-module.html">Home</a>&nbsp;&nbsp;&nbsp;</th>

  <!-- Tree link -->
      <th>&nbsp;&nbsp;&nbsp;<a
        href="module-tree.html">Trees</a>&nbsp;&nbsp;&nbsp;</th>

  <!-- Index link -->
      <th>&nbsp;&nbsp;&nbsp;<a
        href="identifier-index.html">Indices</a>&nbsp;&nbsp;&nbsp;</th>

  <!-- Help link -->
      <th>&nbsp;&nbsp;&nbsp;<a
        href="help.html">Help</a>&nbsp;&nbsp;&nbsp;</th>

  <!-- Project homepage -->
      <th class="navbar" align="right" width="100%">
        <table border="0" cellpadding="0" cellspacing="0">
          <tr><th class="navbar" align="center"
            ><a class="navbar" target="_top" href="https://github.com/mauriceling/copads">https://github.com/mauriceling/copads</a></th>
          </tr></table></th>
  </tr>
</table>
<table border="0" cellpadding="0" cellspacing="0" width="100%%">
  <tr>
    <td align="left" class="footer">
    Generated by Epydoc 3.0.1 on Sun Feb 25 20:14:29 2018
    </td>
    <td align="right" class="footer">
      <a target="mainFrame" href="http://epydoc.sourceforge.net"
        >http://epydoc.sourceforge.net</a>
    </td>
  </tr>
</table>

<script type="text/javascript">
  <!--
  // Private objects are initially displayed (because if
  // javascript is turned off then we want them to be
  // visible); but by default, we want to hide them.  So hide
  // them unless we have a cookie that says to show them.
  checkCookie();
  // -->
</script>
</body>
</html>
